samples/dnn/mask_rcnn.cpp

```diff
@@ -0,0 +1,200 @@
+// Deep learning based Object Detection and Instance Segmentation using Mask R-CNN
+// VIDEO 360 DEMO: https://www.youtube.com/watch?v=0tU8991QgE8
```

Comment:
(my personal opinion) i do not want to see this kind of links in the code. i think video links should be only in documentation and devs should be selective and refuse low quality videos.

Reply:
I had similar opinion about the linked video too. It would be great if you could link in the code to the blog https://www.learnopencv.com/deep-learning-based-object-detection-and-instance-segmentation-using-mask-r-cnn-in-opencv-python-c/   
It explains its usage and has a good results video which was created after hours of effort on different kinds of objects.

Reply:
@sunitanyk  and @sturkmen72 I removed the video from the code.
Thanks for the suggestion.

You can use the video https://www.youtube.com/watch?v=0tU8991QgE8 on the blog. It is a pleasure to contribute to your excellent work.